modules/configuring-vsphere-connection-settings.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-connection-settings_{context}"]
+= Configuring the vSphere connection settings
+
+[role="_abstract"]
+Modify the following vSphere configuration settings:
+
+* vCenter network address
```

Comment:
suggest we put `vCenter address` here, I did not see `vCenter network address` in page vSphere connection configuration